specification/columns/skudetails.md

```diff
@@ -0,0 +1,51 @@
+# SKU Details
+
+The **SKU Details** column represents a list of relevant properties shared by all resources having the same [**SKU ID**](#skuid) which are not captured in another FOCUS column. These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics.
+
+
+The _SkuDetails_ column adheres to the following requirements:
+
+* The properties (both key and value) contained in the _SkuDetails_ column MUST be shared across all resources having the same _SkuId_.
```

Comment:
We discussed this in Maintainers meeting on 2024-08-12 and the general consensus was that "no data is better than bad data" so we're leaving this as a MUST even if that means that CSPs have no content in this on day 1 because they need to curate what goes into it instead of pumping something that they already have (e.g. Additional_Info in Azure) which may not be a great fit.